delivery_mail_templates/__manifest__.py

```diff
@@ -4,7 +4,7 @@
     'name': "Delivery Custom QWeb Email Templates",
     'version': '15.0.1.1.0',
     'summary': 'Delivery templates using QWeb inheritance',
-    'license': 'AGPL-3',
+    'license': 'LGPL-3',
     'author': "Focusate",
     'website': "http://www.focusate.eu",
     'category': 'Inventory/Delivery',
```

Comment:
```suggestion
    'category': 'Inventory/Discuss',
```

Reply:
Delivery is more meaningful as we can be explicit about part of inventory here.